great_tables/_data_color/base.py

```diff
@@ -167,6 +166,13 @@ def data_color(
     ```
     """
 
+    try:
```

Comment:
mizani moved inside function, so if you don't have it, it doesn't fail on import